assets/scss/components/_dropdown.scss

```diff
@@ -115,6 +115,10 @@ $spaceFromTrigger: $space-3 + $triangleSize;
 	-webkit-transform: translateX(-50%);
 }
 
+.dropdownMenu {
+	width: max-content;
```

Comment:
Do we need to set a fallback for Edge here?

Reply:
I can't think of a good fallback to set. Something like
```
@supports not (width: max-content) {
  .dropdownMenu {
    white-space: nowrap;
  }
}
```
Could lead to text overflowing out of the container in some cases. It's probably less weird to not have a fallback and just let the items wrap "too soon".